src/parser/index.ts

```diff
@@ -164,13 +164,24 @@ const removeTitle = (document: string) => {
 };
 
 /**
- * Cuts a string down to the specified length if needed
+ * Cuts a string down to the discord character limit and corrects missing codeblock backticks if needed
  * Appends '...' if the string was trimmed
+ * Appends '...```' if the codeblock backticks were corrected
  * @param {string} document
  * @param {number} length
  */
-const truncateString = (document: string, length: number) => {
-    return document.length > length ? document.slice(0, length - 3) + '...' : document;
+const truncateString = (document: string) => {
```

Comment:
Can we keep the `MAX_LENGTH` as an input param? Since this is in the parent package, I don't think this function should be specific to the discord character limit, it should work for any character limit provided.